packages/arb-node-core/challenge/faulty_core.go

```diff
@@ -0,0 +1,67 @@
+package challenge
```

Comment:
I haven't looked yet, but if this is only used for tests inside the `challenge` package, it should get an `_test` at the end of the filename. If it's used elsewhere, it's good as is

Reply:
It is used elsewhere, as it has the challenger itself.

Reply:
Oh nevermind I thought you were talking about the package, not the file name. Fixed.